matrix/api-reference.md

```diff
@@ -12,7 +16,7 @@ As a shared service, there are limitations on requests, maximum distances, and n
 
 The following limitations are currently in place.
 
-* `max_locations` is  50 for `one_to_many`, `many_to_one`, and `many_to_many` requests.
+* `max_locations` is  50 for `one_to_many`, `many_to_one`, `many_to_many` and `sources_to_targets` requests.
```

Comment:
Might want to clarify what 50 locations means for sources and targets - I assume it is 50 sources and 50 targets max, correct?